securing_apps/topics/cli-apps/cli-apps.adoc

```diff
@@ -0,0 +1,137 @@
+[[_kcinit]]
+
+== Securing Command Line Applications
+
+Many applications need a command line interface to administer and configure themselves.
+If these command line interfaces are invoking on back-end admin REST interfaces that
+are secured by {project_name}, they need a way to obtain a token to invoke on them.
+This chapter discusses some utilities that {project_name} has that you can
+use to secure your command line interfaces.
+
+The initial and standard way to obtain a token for a command line application
+is to prompt the user for username and password and invoke on the <<_resource_owner_password_credentials_flow, Resource Owners Credential Grant>>
+ OAuth endpoint that {project_name} has.  While a completely acceptable solution,
+there are some disadvantages to this approach.  The obvious one is that you'll have to
```

Comment:
...you will...
Contractions are generally not preferred in tech docs. I won't point out others, but it is a good practice not to use them in this style of writing.

Reply:
I think I remember O'Reilly telling me that a afew times years ago :)